File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/settlement/repository/TransactionSettlementRepository.java
package band.gosrock.domain.domains.settlement.repository;


import band.gosrock.domain.domains.settlement.domain.TransactionSettlement;
import java.util.List;
import org.springframework.data.repository.CrudRepository;

public interface TransactionSettlementRepository
        extends CrudRepository<TransactionSettlement, Long> {
    List<TransactionSettlement> findByEventId(Long eventId);

    void deleteByEventId(Long eventId);
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/settlement/repository/TransactionSettlementRepository.kt
package band.gosrock.domain.domains.settlement.repository

import band.gosrock.domain.domains.settlement.domain.TransactionSettlement
import org.springframework.data.repository.CrudRepository

interface TransactionSettlementRepository : CrudRepository<TransactionSettlement, Long> {
    fun findByEventId(eventId: Long): List<TransactionSettlement>
    fun deleteByEventId(eventId: Long)
}
